tests/jballerina-debugger-integration-test/src/test/java/org/ballerinalang/debugger/test/adapter/evaluation/ExpressionEvaluationTest.java

```diff
@@ -105,6 +105,25 @@ public void newConstructorEvaluationTest() throws BallerinaTestException {
         debugTestRunner.assertExpression(context, "new Location(\"New York\",\"USA\")", "Location", "object");
     }
 
+    @Override
+    @Test
+    public void builtInNameReferenceEvaluationTest() throws BallerinaTestException {
+        // basic, simple types
+        debugTestRunner.assertExpression(context, INT_TYPE_DESC, "int", "typedesc");
+        debugTestRunner.assertExpression(context, FLOAT_TYPE_DESC, "float", "typedesc");
+        debugTestRunner.assertExpression(context, DECIMAL_TYPE_DESC, "decimal", "typedesc");
+        debugTestRunner.assertExpression(context, BOOLEAN_TYPE_DESC, "boolean", "typedesc");
+        // basic, sequence types
+        debugTestRunner.assertExpression(context, STRING_TYPE_DESC, "string", "typedesc");
+        // other types
+        debugTestRunner.assertExpression(context, BYTE_TYPE_DESC, "byte", "typedesc");
+        debugTestRunner.assertExpression(context, JSON_TYPE_DESC, "json", "typedesc");
+        debugTestRunner.assertExpression(context, ANY_TYPE_DESC, "any", "typedesc");
+        debugTestRunner.assertExpression(context, ANYDATA_TYPE_DESC, "anydata", "typedesc");
+        debugTestRunner.assertExpression(context, NEVER_TYPE_DESC, "never", "typedesc");
+        debugTestRunner.assertExpression(context, PARENTHESISED_TYPE_DESC, "int", "typedesc");
```

Comment:
Don't we need to add parenthesis here with `int`, `(int)` ?

Reply:
@lnash94 no we don't, since in here "int" is the expected type even if the expression is having parenthesis (as the parenthesis is only used for the grouping purposes)